zuul.d/project-templates.yaml

```diff
@@ -326,151 +326,181 @@
             vars:
-              ansible_test_collections: true
               ansible_test_integration_targets: "iosxr_.*"
             voting: false
         - ansible-test-network-integration-iosxr-network_cli-python37-scenario01:
             vars:
-              ansible_test_collections: true
               ansible_test_integration_targets: "iosxr_.*"
             voting: false
         - ansible-test-network-integration-iosxr-network_cli-python37-scenario02:
             vars:
-              ansible_test_collections: true
               ansible_test_integration_targets: "iosxr_.*"
             voting: false
-        - ansible-test-network-integration-iosxr-netconf-python37-stable29:
+        - ansible-test-network-integration-iosxr-network_cli-python37-stable210-scenario01:
             vars:
-              ansible_test_collections: true
               ansible_test_integration_targets: "iosxr_.*"
             voting: false
-        - ansible-test-network-integration-iosxr-network_cli-python37-stable29-scenario01:
+        - ansible-test-network-integration-iosxr-network_cli-python37-stable210-scenario02:
             vars:
-              ansible_test_collections: true
               ansible_test_integration_targets: "iosxr_.*"
             voting: false
-        - ansible-test-network-integration-iosxr-network_cli-python37-stable29-scenario02:
+        - ansible-test-network-integration-iosxr-network_cli-python37-stable29-scenario01:
             vars:
-              ansible_test_collections: true
               ansible_test_integration_targets: "iosxr_.*"
             voting: false
-        - ansible-test-network-integration-iosxr-netconf-python38:
+        - ansible-test-network-integration-iosxr-network_cli-python37-stable29-scenario02:
             vars:
-              ansible_test_collections: true
               ansible_test_integration_targets: "iosxr_.*"
             voting: false
         - ansible-test-network-integration-iosxr-network_cli-python38-scenario01:
             vars:
-              ansible_test_collections: true
               ansible_test_integration_targets: "iosxr_.*"
             voting: false
         - ansible-test-network-integration-iosxr-network_cli-python38-scenario02:
             vars:
-              ansible_test_collections: true
+              ansible_test_integration_targets: "iosxr_.*"
+            voting: false
+        - ansible-test-network-integration-iosxr-network_cli-python38-stable210-scenario01:
+            vars:
+              ansible_test_integration_targets: "iosxr_.*"
+            voting: false
+        - ansible-test-network-integration-iosxr-network_cli-python38-stable210-scenario02:
+            vars:
               ansible_test_integration_targets: "iosxr_.*"
             voting: false
         - build-ansible-collection:
             required-projects:
               - name: github.com/ansible-collections/ansible.netcommon
     gate:
       queue: integrated
-      jobs:
```

Comment:
unfortunately we cannot do this here, having non-voting jobs in gate pipeline is just a waste of resources